### DISCUSSION TITLE
Getting envelope timestamp from outgoing documents

### DISCUSSION THREAD

**Discussion by machnst1** [OxalisCommunity/oxalis#636] [General]
Hello,
During the last webinar you said that you could share some example on how to grab the timestamp from the envelope AS4 envelope.
Looking forward to get it.
Thanks
//Stefan Machnow, Opuscapita

**Comment by aaron-kumar** [comment]
@machnst1  Please refer: https://github.com/OxalisCommunity/oxalis/wiki/Peppol-Reporting
